src/Form.jsx

```diff
@@ -207,7 +204,7 @@ class Form extends React.Component {
       formfield = <Fields.CheckBoxGroup {...common} />;
     }
     else if (Type === "image") {
-      formfield = <Fields.Image {...common} />;
+      formfield = <Fields.Image {...common} defaultImagePath={field.defaultImagePath} />;
```

Comment:
why not put this in `common` too?

Reply:
ah i thought `common` is for common props that all those field types share.